src/main/python/hydro_sphere/hydro_sphere.py

```diff
@@ -20,8 +22,19 @@
 args = parser.parse_args()
 
```

Comment:
Nice change Muaaz.
Please fix the minor failure in build .
https://travis-ci.org/lake-lerna/HydroSphere/builds/137272954